tests/unit/api/base_materialize_table_test.py

```diff
@@ -16,7 +16,7 @@
 BaseFeatureOrTargetTableT = TypeVar("BaseFeatureOrTargetTableT", bound=ApiObject)
 
 
-class BaseMaterializedTableApiTest(Generic[BaseFeatureOrTargetTableT]):
+class BaseMaterializedTableApiTest:
```

Comment:
With this `Generic` class, all the test subclasses inheriting this base class are skipped.
```
tests/unit/api/test_historical_feature_table.py:11
  /Users/senbonggee/Workspace/featurebyte/tests/unit/api/test_historical_feature_table.py:11: PytestCollectionWarning: cannot collect test class 'TestHistoricalFeatureTable' because it has a __new__ constructor (from: tests/unit/api/test_historical_feature_table.py)
    class TestHistoricalFeatureTable(BaseMaterializedTableApiTest[HistoricalFeatureTable]):
```